### ISSUE TITLE
RE4R demo hangs entire desktop on startup

### ISSUE THREAD

**Issue by mmbossoni** [HansKristian-Work/vkd3d-proton#1601]
When launching RE4R it will freeze my entire PC for a while then eventually it will unblock.
Sometimes though it doesn't unblock

I can see this message on dmesg
[drm:nv_drm_prime_fence_context_create_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00002600] Failed to allocate fence signaling event

### Software information
Resident Evil 4 Remastered demo

### System information
AMD Ryzen 7 5700X
- GPU: NVIDIA GeForce GTX 1070
- Driver:535.54.03
- Wine version: ProtonGE 8-4
- VKD3D-Proton version: Latest built from git (72dc32d139a1f64319a5ae230ba390f0971eef54)

### Log files
[steam-2231770.log](https://github.com/HansKristian-Work/vkd3d-proton/files/11857981/steam-2231770.log)

**Comment by Blisto91** [comment]
@mmbossoni Could i get you to retest this with latest vkd3d-proton and driver?

**Comment by mmbossoni** [comment]
I got rid of my NVIDIA card, so it won't be possible anymore, sorry.
Don't know if that was related to memory pressure or GPU being at 99% almost all the time but my problems aligned with most people on Proton reports.
As long as I tuned down textures to low to not get near 6gb utilization and kept frames locked game mostly worked.
Sometimes I got the fence error messages or XID events.
But I don't have the card anymore.
